src/intl/en/page-ethereum-history-founder-and-ownership.json

```diff
@@ -0,0 +1,65 @@
+ "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-6": "<strong>July 30, 2015:</strong> The founding team officially launch the Ethereum mainnet by mining the genesis block. This marks the birth of the Ethereum network.",
+"page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-7": "<strong>March 14, 2016:</strong> The Ethereum community implement \"Homestead,\" the first planned upgrade. This signals that Ethereum was ready for mainstream adoption.",
+"page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-8": "We had this notion on the Ethereum project that we'd only really ever get one shot at it, it was really one and done, so we had to get it right.",
+"page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-9": "Ethereum Co-founder",
+"page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-10": "Ethereum's launch marked a major milestone in blockchain technology. It introduced smart contracts and created a platform for decentralized applications.",
+"page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-11": "You can <genesisBlock>always view the Ethereum genesis block</genesisBlock> preserving the moment when Ethereum first came to life.",
+"page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-12": "See the complete list of Ethereum upgrades",
+"page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-13": "Check out the future Ethereum roadmap",
+"page-ethereum-history-founder-and-ownership-who-owns-and-runs-ethereum-now-description-1": "One of the most unique aspects of Ethereum is its ownership structure, or more accurately, its lack of traditional ownership. Unlike a regular company, Ethereum:",
+"page-ethereum-history-founder-and-ownership-who-owns-and-runs-ethereum-now-description-2": "<strong>has no CEO or central authority</strong>",
+"page-ethereum-history-founder-and-ownership-who-owns-and-runs-ethereum-now-description-3": "is <strong>not controlled by any single entity</strong> or organization",
+"page-ethereum-history-founder-and-ownership-who-owns-and-runs-ethereum-now-description-4": "<strong>does not have shareholders</strong> in the traditional sense",
+"page-ethereum-history-founder-and-ownership-who-owns-and-runs-ethereum-now-description-5": "Instead, Ethereum works as a decentralized network. It's <strong>maintained by thousands of independent computers</strong> (nodes) around the world. This decentralized model is at the core of Ethereum's design and value.",
+"page-ethereum-history-founder-and-ownership-who-owns-and-runs-ethereum-now-description-6": "Several key groups help with Ethereum's ongoing development and governance:",
+"page-ethereum-history-founder-and-ownership-ethereum-foundation": "1. The Ethereum Foundation",
```

Comment:
We should remove the numerals from the strings and use an ordered list in the React page instead

Reply:
Can still keep them as h3's to my knowledge with pattern:

```html
<ol>
  <li>
    <h3>Header</h3>
  </li>
  <li>
    <h3>Another header</h3>
  </li>
</ol>
```

Please correct me if I'm wrong